### PR TITLE
[DOCFIX] modify docs/cn/advanced/Alluxio-Storage-Management.md

### PR DESCRIPTION
**Pull request by lxcnju**
modify docs/cn/advanced/Alluxio-Storage-Management.md

### CODE REVIEW

**Comment by lxcnju** [comment]
Jenkins, test this please

File: docs/cn/advanced/Alluxio-Storage-Management.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ priority: 0
 * 内容列表
 {:toc}
 
-Alluxio管理Alluxio workers的本地存储，包括内存，来充当分布式缓冲缓存区。这个在用户应用程序和各种底层存储之间的快速数据层可以很大程度上改善I/O性能。
+Alluxio 管理了 Alluxio workers 节点的包括内存在内的本地存储，来充当分布式缓冲缓存区。这个在用户应用程序和各种底层存储之间的快速数据层可以很大程度上改善I/O性能。
```
Comment (LuQQiu) [inline docs/cn/advanced/Alluxio-Storage-Management.md]: In our convention, we do not add spaces between Chinese words and English words.
Could you change `Alluxio 管理了 Alluxio workers 节点的包括内存在内的本地存储` to `Alluxio管理了Alluxio workers节点的包括内存在内的本地存储`?

**Comment by LuQQiu** [comment]
@lxcnju Could you change the title of this PR to
`[DOCFIX] modify docs/cn/advanced/Alluxio-Storage-Management.md` ? Thanks

**Comment by lxcnju** [comment]
Thanks for your advice @LuQQiu

File: docs/cn/advanced/Alluxio-Storage-Management.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ priority: 0
 * 内容列表
 {:toc}
 
-Alluxio管理Alluxio workers的本地存储，包括内存，来充当分布式缓冲缓存区。这个在用户应用程序和各种底层存储之间的快速数据层可以很大程度上改善I/O性能。
+Alluxio 管理了Alluxio workers节点的包括内存在内的本地存储，来充当分布式缓冲缓存区。这个在用户应用程序和各种底层存储之间的快速数据层可以很大程度上改善I/O性能。
```
Comment (LuQQiu) [inline docs/cn/advanced/Alluxio-Storage-Management.md]: Could you remove the extra space between `Alluxio` and `管理`?

**Comment by lxcnju** [comment]
Sorry, I did not remove all of the blanks between chinese characters and english ones yesterday. Now, I did it just some seconds before. Thanks again! @LuQQiu

**Pull request merged**